components/fieldlabel/index.css

```diff
@@ -108,6 +104,7 @@ governing permissions and limitations under the License.
 .spectrum-FieldLabel--left,
 .spectrum-FieldLabel--right {
   display: inline-block;
+  vertical-align: top;
```

Comment:
Note: this is moved from the main `.spectrum-FieldLabel` class because it only applies with inline-block or previously in a table cell; on the main class it does not have an affect—this removes the postcss warning `Property is ignored due to the display. With 'display: block', vertical-align should not be used.`